documentation/book/con-requests-kafka-bridge.adoc

```diff
@@ -5,21 +5,31 @@
 [id='con-requests-kafka-bridge-{context}']
 = Requests to the Kafka Bridge
 
-Specify data formats and HTTP headers to ensure valid requests are submitted to the Kafka Bridge. 
+Specify data formats and HTTP headers to ensure valid requests are submitted to the Kafka Bridge.
+
+API request and response bodies are always encoded as JSON.
 
 == Content Type headers
 
-API request and response bodies are always encoded as JSON.
+A `Content-Type` header must be submitted for all requests (apart from the exception described below). 
+
+Consumer operations (`/consumers` endpoints) and producer operations (`/topics` endpoints) require different `Content-Type` headers.
+
+.Content-Type headers for consumer operations
+
+Regardless of the link:#embedded-data-format[embedded data format], `POST` requests for consumer operations must provide the following `Content-Type` header if the request body contains data: 
 
-* When performing consumer operations, `POST` requests must provide the following `Content-Type` header if there is a non-empty body:
-+
 [source,http,subs=+quotes]
 ----
 Content-Type: application/vnd.kafka.v2+json
 ----
 
-* When performing producer operations, `POST` requests must provide `Content-Type` headers specifying the desired _embedded data format_, either `json` or `binary`, as shown in the following table.
-+
+If the `POST` request body is empty, requests sent with a `Content-Type` header will fail because the OpenAPI Specification (OAS) will not validate the request.
```

Comment:
Why not mention this upfront instead of using the note in brackets?

Reply:
Well, this qualification applies to consumer operations only, so it's not quite accurate to include it at the start (where I have the note in brackets). I would need to restructure this section.

Reply:
After some thought, I moved the sentence to the start and reworded again:

## Content Type headers

A `Content-Type` header must be submitted with all requests. The only exception is for consumer operations where the `POST` request body is empty. In this case, requests sent with a `Content-Type` header will fail because the OpenAPI Specification (OAS) does not validate the request.